app/views/kudos/_kudos.html.erb

```diff
@@ -1,4 +1,5 @@
-<% # expects local variables kudos, guest_kudos_count, commentable %>
+<%# expects local variables kudos, guest_kudos_count, commentable %>
```

Comment:
Could you add the space back between `%` and `#`?

Reply:
Done, but may I ask why?
erb linter was complaining about that space.
I also found [several](https://stackoverflow.com/questions/2774841/best-way-to-add-comments-in-erb#comment2811714_2774858) [posts](https://stackoverflow.com/a/3901665) saying that propper comment syntax is `<%#`

And thanks for the skin tips! Had to add all of them by copying, but all looks good.

Reply:
Sure! I don't know if there's a particular reason we chose it, but the space is our standard -- we have one use without the space and 260 with it -- so I just asked for consistency's sake. 

And I'm glad the skin tips worked!